app/models/cms/builtin.rb

```diff
@@ -135,20 +135,17 @@ class Page < CMS::Builtin
     def to_xml(options = {})
       xml = options[:builder] || Nokogiri::XML::Builder.new
 
-      xml.builtin_page do |x|
+      xml.__send__(CMS::XML::CLASS_TAG.call(self.class)) do |x|
         unless new_record?
           xml.id id
           xml.created_at created_at.xmlschema
           xml.updated_at updated_at.xmlschema
         end
-
-        # x.title title
-        x.system_name system_name
-        x.liquid_enabled liquid_enabled
-
+        x.title title
+        x.layout_id layout_id
         unless options[:short]
-          x.draft draft
-          x.published published
+          x.draft { |node| node.cdata draft }
+          x.published { |node| node.cdata published }
```

Comment:
has to be tested if this works with CDATA including HTML content

Reply:
I added tests for:

- HTML inside CDATA
- CDATA inside CDATA